### PR TITLE
[#1913] toggle nation in configuration(connects #1913)

### PR DESCRIPTION
**Pull request by yuviii**

### CODE REVIEW

**Comment by yuviii** [comment]
@paulbert @lmmrssa @dogi Please review the fixes.

File: src/app/community/community.component.ts [hunk 1]
```diff
@@ -136,12 +136,25 @@ export class CommunityComponent implements OnInit, AfterViewInit, OnDestroy {
...
+          });
+          break;
         case 'accept':
           forkJoin([
             // When accepting a registration request, add learner role to user from that community/nation,
             this.unlockUser(community),
             // update registration request to accepted
-            this.couchService.put('communityregistrationrequests/' + communityId, { ...community, registrationRequest: 'accepted' })
+            this.couchService.put('communityregistrationrequests/' + communityId,
+            { ...community, registrationRequest: 'accepted', hide: true })
```
Comment (paulbert) [inline src/app/community/community.component.ts]: This addition doesn't seem to do anything.  Could you explain or remove?

**Comment by paulbert** [comment]
#1913

**Comment by yuviii** [comment]
not relevant anymore

**Pull request closed without merging**